locallib.php

```diff
@@ -161,34 +223,31 @@ function hvp_add_editor_assets($id = null, $mformid = null) {
     $url = \mod_hvp\view_assets::getsiteroot() . '/mod/hvp/';
     $url = '/' . preg_replace('/^[^:]+:\/\/[^\/]+\//', '', $url);
 
-    // Make sure files are reloaded for each plugin update.
-    $cachebuster = \hvp_get_cache_buster();
-
     // Add editor styles.
     foreach (H5peditor::$styles as $style) {
-        $assets['css'][] = $url . 'editor/' . $style . $cachebuster;
+        $assets['css'][] = generate_css_url('editor/' . $style)->out();
```

Comment:
Have we also lost $cachebuster along the way here?

Reply:
The pluginfile adds it as an etag, which I believe is used as a version for caching.

Reply:
Yeah, I see now that etag isn't sufficient. Will change things.

Reply:
The css on the other hand does need a cache buster path component or query param:

This is 2 page loads, with a cache purge between and they are the same:

![image](https://user-images.githubusercontent.com/187449/206367576-da62c09c-2c27-4d83-89b7-90dff8ae6257.png)

Also do you know why each url is requested twice?

Reply:
The code generates an iframe. The assets are requested by both the iframe and by the main page.

Reply:
The css does have a cache buster, I think your browser isn't showing it @brendanheywood 
![image](https://user-images.githubusercontent.com/17095477/206573734-88b15d3d-cd7e-4856-bd42-754b3e57ba33.png)

Reply:
The CSS does seem to already have a cache buster but @jaypha  is correct in that its being called from `view.php` and `h5p.js` so probably why it isn't cached from the initial load but subsequent loads use the respective cache for each calling path

![image](https://user-images.githubusercontent.com/17095477/206574256-c100d84b-2fd1-4bb9-af81-e7bfa444595e.png)